test/binCases/stats/single-config/test.js

```diff
@@ -0,0 +1,30 @@
+/*globals it*/
+module.exports = function validateBinExec(cmd, args, opts) {
```

Comment:
Awesome got my todo for tonight. Thank you for this. Much more clear on how to tackle.